### DISCUSSION TITLE
[OFFTOPIC] mod-autobalance: need feedbacks from live servers

### DISCUSSION THREAD

**Discussion by Yehonal** [azerothcore/azerothcore-wotlk#3948] [General]
Hi there

I'm going to work on this module in the near future and I need feedbacks from live server to improve its features

Please contact me on discord at Yehonal#4264

If you will help me I can give you free access to some premium modules that I'll release in the next weeks (such as Mythic+ and improved levelling)

**Comment by chamorrorenzo** [comment]
Hi, what exactly are you looking for on the feedbacks from the module? Maybe I can help with that
I have so much free time to waste on the next weeks

**Comment by BRNKR** [comment]
Dungeons alone should be challenging like with 2 or more. i think the difficulty is not the same.

i dont know if it is a autobalance problem. but when i die in dungeon and reenter it, it is resetted. or is it a lfg solo bug?

**Comment by AciDCooL** [comment]
Heya, i've been using this while lvling with my wife and have to say this is a pretty good concept and fit's right in with our needs. But it needs some more tuning though.
I mean I used this while going into Stockades, and it was a blast at lvl 20, never have I ever powerleveled so quickly as with this module. And of course now I know this can be fixed with (AutoBalance.DungeonScaleDownXP) but It should be the default instead of earning wild amounts of XP right of the bat, unless that is the intended goal of the module.
Also I tuned the default 0.5 now to 0.3 and still have to test the difficulty, but with full heirloom gear 0.5 felt really easy. I don't need it to be darksouls heavy but still need to be able to solo it. Trying to find the sweetspot here.

**Comment by Yehonal** [comment]
> Dungeons alone should be challenging like with 2 or more. i think the difficulty is not the same.
> 
> i dont know if it is a autobalance problem. but when i die in dungeon and reenter it, it is resetted. or is it a lfg solo bug?

I think this problem it's strictly related to lfg. Could you try with autobalance disabled please?

**Comment by Yehonal** [comment]
> Heya, i've been using this while lvling with my wife and have to say this is a pretty good concept and fit's right in with our needs. But it needs some more tuning though.
> I mean I used this while going into Stockades, and it was a blast at lvl 20, never have I ever powerleveled so quickly as with this module. And of course now I know this can be fixed with (AutoBalance.DungeonScaleDownXP) but It should be the default instead of earning wild amounts of XP right of the bat, unless that is the intended goal of the module.
> Also I tuned the default 0.5 now to 0.3 and still have to test the difficulty, but with full heirloom gear 0.5 felt really easy. I don't need it to be darksouls heavy but still need to be able to solo it. Trying to find the sweetspot here.


The DungeonScaleDownXP should be enabled by default, Indeed

The time you need to complete stockade alone should be the same completing it with 5 people. Do you have the possibility to test it?

**Comment by chamorrorenzo** [comment]
Hi yehonal, I cant find you on discord
I'll be happy to help

Discord: hongito#3370

**Comment by AciDCooL** [comment]
> The DungeonScaleDownXP should be enabled by default, Indeed
> 
> The time you need to complete stockade alone should be the same completing it with 5 people. Do you have the possibility to test it?

Ok I tested it with 0.3 (im lvl 37, mobs are around 1400hp), and now it feels more like pull a pack or two and kill them and then move on. With 0.5 I can pull 1 entire corridor with ease and lvl up like crazy.
Not sure how this holdup against heroic dungeons later on, but ill find out this weekend when me and the misses have some more time.

**Comment by disclosurez** [comment]
ability to set different inflection points per mapid would be nice.

**Comment by Yehonal** [comment]
I'll start to work on it again in the near future

So, stay tuned!

Your feedback will be very appreciated

**Comment by steenburgh** [comment]
I've been running a server with ~7 active players running TBC content + here's my feedback:
* I can definitely get behind a per-map inflection point
  * I'd like to be able to set it based on map level. Some players have expressed interest in soloing low level instances, but complained that level 70 5-mans were too easy.
  * As far as setting a single inflection point for a single map, I think this would work best if I could set this in-game via a command that so I don't have to do any database work and/or restart the server.
* The biggest issue we've found with AutoBalance is creatures with CC/fear abilities. This makes running some instances hell. I don't have a great idea of how to fix this, but I'm curious what ideas other folks might have. In particular, the trash in a couple of Auchindoun 5-mans really likes to fear/charm players. I could also imagine this would become a problem with things like Lord Marrowgar's 'Bone Spike Graveyard' and Sindragosa's 'Ice Tomb', but we aren't running that content yet.
* The second biggest issue we've found with autobalance is bosses that require off tanks, bosses that hit x number of players such as Prince Malchezaar's `Enfeeble`. I think this would be hard if not impossible to fix, but again just want to get it out there to see if folks have ideas.

Also, here are the settings I've been using for my server. We've geared past 5-man heroics and are currently starting to run TBC 10/25 mans, so this means Kara, Gruul's Lair + Magtheridon.
 * We regularly run 5-mans with 1-3 players, and this makes them sufficiently challenging
 * We regularly run 10/25 mans with 4-6 players, and this makes it sufficiently challenging
 * Soloing 10/25 mans is close to impossible with these settings. For our use case, this is perfect as we don't want folks to solo them anyways.

```
AutoBalance.InflectionPoint=0.4
AutoBalance.InflectionPointHeroic=0.4

AutoBalance.InflectionPointRaid10M=0.35
AutoBalance.InflectionPointRaid10MHeroic=0.35

AutoBalance.InflectionPointRaid25M=0.35
AutoBalance.InflectionPointRaid25MHeroic=0.35

AutoBalance.InflectionPointRaid=0.35
AutoBalance.InflectionPointRaidHeroic=0.35
```

**Comment by BarbzYHOOL** [comment]
Good suggestions, for spells we had started a work in a PR by Dunjeon but it got stopped sadly

Being able to set the balance per map ID was something i suggested too (and rename to "mod-balance" then? xD) so we could completely tune every dungeon one by one, would be crazy good

**Comment by steenburgh** [comment]
If you're talking about modifying spells, we can probably take some hints from the private server, [TBC5MAN](https://tbc5man.com/)

**Comment by Parkan1986** [comment]
Question to developers:Is azerothcore version of autobalance can scale level and stats of mobs in open world? 
If not, is it possible to create such feature?

**Comment by WotlkCasually** [comment]
just started using this module in my own wotlk project and it feels really good, will give u feedback soon :)
